packages/doenetml-worker-javascript/src/components/abstract/BaseComponent.js

```diff
@@ -1684,11 +1684,12 @@ export default class BaseComponent {
         return { success: true, desiredVariant };
     }
 
+    // TODO: not functioning properly, but also not called anywhere
     async copyDoenetMLToClipboard({ actionId }) {
```

Comment:
Why are we keeping the function if its broken and not used?

Reply:
Oops. I fixed it and forgot to remove the comment.

I also forgot that it was an "action" so can be called from DoenetML despite the fact that it isn't called anywhere in the codebase itself. Will need to document this.